tests/e2e/performance/test_pvc_creation_performance.py

```diff
@@ -62,23 +93,40 @@ def base_setup(
     @pytest.mark.usefixtures(base_setup.__name__)
     def test_pvc_creation_measurement_performance(self, teardown_factory, pvc_size):
         """
-        Measuring PVC creation time is less than 3 seconds
-        """
-        log.info('Start creating new PVC')
+        The test measures PVC creation times for sample_num volumes
+        (limit for the creation time for pvc is defined in accepted_create_time)
+        and compares the creation time of each to the accepted_create_time ( if greater - fails the test)
+        If all the measures are up to the accepted value
+        The test calculates .... difference between creation time of each one of the PVCs and the average
+        is not more than Accepted diff ( currently 5%)
 
-        pvc_obj = helpers.create_pvc(
-            sc_name=self.sc_obj.name, size=pvc_size
-        )
-        helpers.wait_for_resource_state(pvc_obj, constants.STATUS_BOUND)
-        pvc_obj.reload()
-        teardown_factory(pvc_obj)
-        create_time = helpers.measure_pvc_creation_time(
-            self.interface, pvc_obj.name
-        )
-        logging.info(f"PVC created in {create_time} seconds")
-        if create_time > 3:
+        Args:
+            teardown factory: A fixture used when we want a new resource that was created during the tests
+                               to be removed in the teardown phase.
+            pvc_size: Size of the created PVC
+         """
+        num_of_samples = 3
```

Comment:
@ypersky1980  Is this number is going to be fixed or we plan to change randomly, if fixed to minimal number like 3 PVCs then its good to use measure_pvc_creation_time() fun, but if the plan is to create n numbers i.e. more than 10 then would suggest to use https://github.com/red-hat-storage/ocs-ci/blob/master/tests/helpers.py#L1169

One more question, what kind of PVC are we testing here ? It would be good to capture the perf for all 4 types of PVC